bootstrap.sh

```diff
@@ -21,13 +21,16 @@ if [[ "${RELEASE}" == "master" || "${RELEASE}" == release-0.1 ]]; then
   set +x
   echo "NOTICE: the trace and pack commands are not available from master. RELEASE must be set to a released version (such as v0.2.0). See https://github.com/crossplane/crossplane-cli/releases for the full list of releases." >&2
   set -x
+  curl -sL -o "${PREFIX}"/bin/kubectl-crossplane-package https://raw.githubusercontent.com/crossplane/crossplane-cli/"${RELEASE}"/bin/kubectl-crossplane-package >/dev/null
```

Comment:
how about the `kubectl-crossplane` command too? that seems to be missing from this list, and maybe is the reason why i keep seeing `kubectl crossplane` results in:
```
Error: unknown command "crossplane" for "kubectl"
Run 'kubectl --help' for usage.
```

Reply:
oh and what about `kubectl-crossplane-krew-wrapper` also?

Reply:
@jbw976 are you familiar with how `kubectl-crossplane-krew-wrapper` works?

Reply:
i cannot say i am really, but I was thinking from this line that *everything* in the bin directory of the repo should be included: https://github.com/crossplane/crossplane-cli/blob/master/bootstrap.sh#L32